ui/imports/shared/controls/ImportKeypairInfo.qml

```diff
@@ -0,0 +1,55 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Rectangle {
+    id: root
+
+    radius: Style.current.radius
+    border.width: 1
+    border.color: Theme.palette.directColor8
+    color: Theme.palette.transparent
+
+    property string title: qsTr("Import keypair to use this account")
+    property string info: qsTr("This account was added to one of your synced devices. To use this account you will first need import the associated keypair to this device.")
+    property string buttonName: qsTr("Import keypair")
```

Comment:
readonly?

Reply:
No, they cannot be readonly cause those are exposed properties, with defined default values, that can be changed from outside.